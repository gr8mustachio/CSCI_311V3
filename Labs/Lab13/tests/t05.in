11
addEdge 0 1
addEdge 1 4
addEdge 1 2
addEdge 2 4
addEdge 2 3
addEdge 3 4
addEdge 3 8
addEdge 4 5
addEdge 4 6
addEdge 5 0
addEdge 6 7
addEdge 7 4
addEdge 8 9
addEdge 9 10
addEdge 10 5
printGraph
longestCycle 4
