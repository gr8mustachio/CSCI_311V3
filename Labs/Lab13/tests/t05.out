1 
4 2 
4 3 
4 8 
5 6 
0 
7 
4 
9 
10 
5 
The length of the longest cycle, on which 4 lies is 6
